cmd/bitxhub/client/governance.go

```diff
@@ -109,6 +109,89 @@ func governanceCMD() cli.Command {
 			roleMgrCMD(),
 			dappMgrCMD(),
 			serviceMgrCMD(),
+			proposalStrategyCMD(),
+		},
+	}
+}
+
+func proposalStrategyCMD() cli.Command {
+	return cli.Command{
+		Name:  "strategy",
+		Usage: "proposal strategy command",
+		Subcommands: cli.Commands{
+			cli.Command{
+				Name:  "all",
+				Usage: "query all proposal strategy",
+				Action: func(ctx *cli.Context) error {
+					receipt, err := invokeBVMContractBySendView(ctx, constant.GovernanceContractAddr.String(), "GetAllProposalStrategy")
+					if err != nil {
+						return fmt.Errorf("invoke BVM contract failed when get all proposal strategy: %w", err)
+					}
+
+					if receipt.IsSuccess() {
+						strategies := make([]*contracts.ProposalStrategy, 0)
+						if err := json.Unmarshal(receipt.Ret, &strategies); err != nil {
+							return fmt.Errorf(err.Error())
+						}
+						printProposalStrategy(strategies)
+					} else {
+						color.Red("get all proposal strategy error: %s\n", string(receipt.Ret))
+					}
+					return nil
+				},
+			},
+			cli.Command{
+				Name:  "update",
+				Usage: "update proposal strategy",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:     "module",
+						Usage:    "module name",
+						Required: true,
+					},
+					cli.StringFlag{
+						Name:     "typ",
+						Usage:    "proposal strategy(SuperMajorityApprove, SuperMajorityAgainst, SimpleMajority or ZeroPermission)",
```

Comment:
SuperMajorityApprove and SuperMajorityAgainst haven't been implemented yet, so it's better not list them